tools/wptrunner/wptrunner/tests/test_hosts.py

```diff
@@ -43,14 +43,14 @@ def test_alignment(self):
 ""","""127.0.0.1   localhost    alias
 192.168.1.1 another_host another_alias
 """
-)
+                     )
 
     def test_multiple_same_name(self):
         # The semantics are that we overwrite earlier entries with the same name
         self.do_test("""127.0.0.1    \tlocalhost  alias
 192.168.1.1 localhost    another_alias""","""192.168.1.1 localhost another_alias
 """
-)
+                     )
```

Comment:
Would it complain about putting the closing paren on the previous line? If not, I'd prefer that.